src/steps/organization/organization.test.ts

```diff
@@ -145,3 +156,297 @@ describe('#fetchOrganizationMembers', () => {
     });
   });
 });
+
+describe('#fetchOrganizationWorkspaces', () => {
+  test('should collect data', async () => {
+    const context = createMockStepExecutionContext<IntegrationConfig>({
+      instanceConfig: integrationConfig,
+    });
+
+    await cacheOrganizationData(context.jobState, [
+      // {
```

Comment:
do we need to keep these comments?

Reply:
Good catch, they were necessary until the previous change and I forgot to remove them after making sure tests work without them. Thanks for marking all of those!